src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write code here
+// search elements
+const field = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+// add style
+field.style.cssText = `
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  `;
+spider.style.position = `relative`;
```

Comment:
You need to calculate spiders position based on it's width and height, and walls width and height respectively